packages/devtools-modules/client/shared/components/splitter/Draggable.js

```diff
@@ -19,24 +19,25 @@ const Draggable = React.createClass({
 
   startDragging(ev) {
     ev.preventDefault();
-    const doc = ReactDOM.findDOMNode(this).ownerDocument;
-    doc.addEventListener("mousemove", this.onMove);
-    doc.addEventListener("mouseup", this.onUp);
+    const elt = ReactDOM.findDOMNode(this);
+    elt.setCapture(true);
```

Comment:
when called on mousedown, all mouse events are then redirected to this element until mouseup (no need to use releaseCapture).

"true" is to retarget events to this element -- this overcomes the iframe issue that was mentioned in the previous code.